app/meson.build

```diff
@@ -440,3 +440,11 @@ executable('UsocketClientSample', usocket_client_sample_sources,
   dependencies: [asan_dep, mtl, libpthread]
 )
 endif
+
+# v4l2 to IP sample app
+executable('V4l2toIPSample', v4l2_to_ip_sample_sources,
+  c_args : app_c_args,
+  link_args: app_ld_args,
+  # asan should be always the first dep
+  dependencies: [asan_dep, mtl, libjson_c, libpcap, libsdl2, libsdl2_ttf, libm, libpthread]
```

Comment:
Please check if `libjson_c, libpcap` are really used in this application

Reply:
> Please check if `libjson_c, libpcap` are really used in this application

Done.